src/theme/index.js

```diff
@@ -1,6 +1,7 @@
 import defaultTheme from './default-theme';
 // see cli actions.js to understand this import
-let userTheme = require('spectacle-user-theme').default;
+// TODO: replace with global theme variable?
```

Comment:
Let's add a ticket to support this comment and paste the ticket link in a code comment underneath this comment.